package.json

```diff
@@ -9,7 +9,9 @@
     "format": "prettier --write contracts/**/*.sol test/**/*.js",
     "format:check": "prettier --check contracts/**/*.sol test/**/*.js",
     "lint": "solhint contracts/**.sol",
-    "deploy": "hardhat deploy"
+    "deploy": "hardhat deploy",
+    "verify": "pnpm verify:marketplace",
```

Comment:
@0xb337r007 I believe we should stick to `npm` in this repo here. Currently codex doesn't use our template, so pnpm is likely not installed on everyone's machines.
Given that `pnpm` is only used for running this script (and not for installing dependencies), we should just stick to npm until the repo moves over to a different tooling.